python_ta/checkers/forbidden_io_function_checker.py

```diff
@@ -48,31 +51,36 @@ class IOFunctionChecker(BaseChecker):
 
     @only_required_for_messages("forbidden-IO-function")
     def visit_call(self, node: nodes.Call) -> None:
-        if isinstance(node.func, nodes.Name):
-            name = node.func.name
-            # ignore the name if it's not a builtin (i.e. not defined in the
-            # locals nor globals scope)
-            if not (name in node.frame() or name in node.root()):
-                scope = node.scope()
-                scope_parent = scope.parent
+        if (
+            name := self._resolve_qualname(node)
+        ) is not None and name in self.linter.config.forbidden_io_functions:
```

Comment:
the `is not None` check is redundant; if `None` is returned it won't be in the `forbidden_io_functions` list